app/src/main/res/values-de/strings.xml

```diff
@@ -1,65 +1,57 @@
 <resources>
-    <string name="app_name">Schlichter Musikplayer</string>
-    <string name="app_launcher_name">Musikplayer</string>
+    <string name="app_name">Schlichter Musik-Player</string>
+    <string name="app_launcher_name">Musik-Player</string>
     <string name="playlist_empty">Deine Playlist ist leer</string>
     <string name="rename_song">Lied umbenennen</string>
     <string name="rename_song_error">Während des Umbenennens ist ein Fehler aufgetreten</string>
     <string name="rename_song_empty">Fülle bitte alle Felder aus</string>
-    <string name="enable_song_repetition">Aktuelles Lied wiederholen</string>
+    <string name="enable_song_repetition">Aktuelles Lied Wiederholen</string>
     <string name="disable_song_repetition">Aktuelles Lied nicht mehr wiederholen</string>
-    <string name="song_repetition_enabled">Wiederholung des aktuellen Lieds aktiviert</string>
-    <string name="song_repetition_disabled">Wiederholung des aktuellen Lieds deaktiviert</string>
+    <string name="song_repetition_enabled">Wiederholung des aktuellen Liedes aktiviert</string>
+    <string name="song_repetition_disabled">Wiederholung des aktuellen Liedes deaktiviert</string>
     <string name="enable_shuffle">Zufällige Wiedergabe aktivieren</string>
     <string name="disable_shuffle">Zufällige Wiedergabe deaktivieren</string>
     <string name="shuffle_enabled">Zufällige Wiedergabe aktiviert</string>
     <string name="shuffle_disabled">Zufällige Wiedergabe deaktiviert</string>
-    <string name="enable_autoplay">Automatisches Abspielen aktivieren</string>
-    <string name="disable_autoplay">Automatisches Abspielen deaktivieren</string>
+    <string name="enable_autoplay">Automatische Wiedergabe aktivieren</string>
+    <string name="disable_autoplay">Automatische Wiedergabe deaktivieren</string>
     <string name="autoplay_enabled">Automatisches Abspielen aktiviert</string>
```

Comment:
If you changed it above, please change this to `Automatische Wiedergabe aktiviert` too!